src/test/java/bio/terra/service/journal/JournalServiceTest.java

```diff
@@ -206,6 +206,55 @@ public void journal_UnwindEntryTest() {
         hasSize(1));
   }
 
+  @Test
+  public void journal_DeleteEntriesByFlightIdTest() {
```

Comment:
Thank you for adding this test!

What do you think about expanding testing to include the following cases:
- Deleting entries when there are none for the specified flight ID (such as if walking back a flight with multiple journaling steps)
- Deleting entries when the `changeMap` includes more than just the `flightId`, since the underlying DB command is unpacking the `mutation` field (IIRC, the `changeMap` is loaded into `mutation`)
- Deleting entries when there are also entries for a different flight ID

Reply:
Thanks @okotsopoulos! I updated the test to include the scenarios you mentioned. In the process, I noticed `journal_UnwindEntryTest` was now the only place in our code calling `journalService.removeJournalEntry` so I took out the unused code.

Reply:
Oh nice!